ultralytics/data/augment.py

```
@@ -2213,10 +2213,16 @@ def _format_img(self, img):
             >>> print(formatted_img.shape)
             torch.Size([3, 100, 100])
         """
+        # if len(img.shape) < 3:
+        #     img = np.expand_dims(img, -1)
+        # img = img.transpose(2, 0, 1)
+        # img = np.ascontiguousarray(img[::-1] if random.uniform(0, 1) > self.bgr and img.shape[0] == 3 else img)
```

Pylint (3.3.7)

[convention] Line too long (113/100)

(C0301)